deployment/terraform/modules/ec2/main.tf

```diff
@@ -347,3 +347,21 @@ resource "aws_elb" "https" {
     unhealthy_threshold = 4
   }
 }
+
+resource "aws_cloudwatch_metric_alarm" "cpu_utilization" {
+  alarm_name                = "cpu-utilization-limit-exceeded"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
```

Comment:
Terraform style guide calls for aligning all equal signs within a block to the longest attribute name which is this one.